src/javascript/app_2/Modules/Page404/Components/Page404.jsx

```diff
@@ -0,0 +1,27 @@
+import React          from 'react';
+import { Link }       from 'react-router-dom';
+import { routes }     from 'Constants/index';
+import { localize }   from '_common/localize';
+import { Icon404 }    from './Icon404.jsx';
+import Button         from '../../../App/Components/Form/button.jsx';
+import ErrorBox       from '../../../App/Components/Elements/ErrorBox';
+
+const Page404 = () => (
+    <div className='page-error-container'>
+        <ErrorBox
+            header={localize('Page not found')}
+            icon={<Icon404 />}
+            message={localize('Sorry, we couldn\'t find the page you are looking for.')}
+        >
+            <Link to={routes.trade}>
```

Comment:
didn't we decide to suggest the closest page to them? e.g. if there's `statement` in the path, suggest `statement` instead of trade, and fallback to trade by default? @morteza-binary @ashkanx

Reply:
It was suggested, but I think we might need an additional design for this. 

For me the Trade button should always be there in `404` to encourage clients back to Trade, but we can add a suggestion message below the button, something along the lines of `Did you mean: so and so`, similar to how Search Engines do it.

Reply:
let's add the `trade` only. looks like New new-design will bring statement and portfolio into the trade page